src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -104,15 +104,15 @@
     public static final String MSG_KEY = "singleline.javadoc";
 
     /**
-     * Specify at-clauses which are ignored by the check.
+     * Specify which are ignored by the check.
```

Comment:
I updated issue title and description to mention that term should be changed to `group of block tags`
please do.